rust/hyperlane-base/src/settings/signers.rs

```diff
@@ -80,17 +86,20 @@ impl BuildableWithSignerConf for hyperlane_ethereum::Signers {
                 let signer = AwsSigner::new(client, id, 0).await?;
                 hyperlane_ethereum::Signers::Aws(signer)
             }
-            SignerConf::CosmosKey { key, .. } => hyperlane_ethereum::Signers::Local(
-                LocalWallet::from(ethers::core::k256::ecdsa::SigningKey::from(
-                    ethers::core::k256::SecretKey::from_be_bytes(key.as_bytes())
-                        .context("Invalid ethereum signer key")?,
-                )),
-            ),
+            SignerConf::CosmosKey { .. } => {
+                bail!("cosmosKey signer is not supported by Ethereum")
+            }
```

Comment:
There seems to be a problem with the validator announce in the cosmos chain. Cosmos Chain also uses Etheruem Signer. @tkporter